client/src/pages/Feed.js

```diff
@@ -108,16 +280,60 @@ const Feed = () => {
         handleFilterModal,
         handleQuit,
         handleLocation,
+        handleOnClose,
+        showFilters,
       }}
     >
       <FeedWrapper>
-        <FilterBox />
-        <Posts filteredPosts={fakePosts} />
-        <SvgIcon
-          src={creatPost}
-          onClick={handleCreatePost}
-          className="create-post"
-        />
+        <LayoutWrapper>
+          <SiderWrapper
+            breakpoint="md"
+            className="site-layout-background"
+            width={290}>
+            <div>
+              <MenuWrapper
+                defaultSelectedKeys={['ALL']}
+                onClick={handleChangeType}
+              >
+                {Object.keys(HELP_TYPE).map((item, index) =>
+                  <Menu.Item key={item}>
+                    {HELP_TYPE[item]}
+                  </Menu.Item>
+                )}
+              </MenuWrapper>
+              <FiltersWrapper>
+                <button
+                  onClick={handleShowFilters} >
+                  <span>
+                    <FiltersIcon />
+                  </span>
+                Filters
+              </button>
+                <FiltersList />
+              </FiltersWrapper>
+            </div>
+            <FiltersSidebar />
+          </SiderWrapper>
+          <ContentWrapper>
+            <HeaderWrapper>
+              <h1>Feed</h1>
+              <button
+                onClick={handleCreatePost}>
+                Create post
```

Comment:
Let's give it a color of black.

current
<img width="230" alt="Screen Shot 2020-05-10 at 8 13 00 PM" src="https://user-images.githubusercontent.com/42228386/81520951-a7a04800-92fa-11ea-9a0a-949c95f1283b.png">

figma
<img width="159" alt="Screen Shot 2020-05-10 at 7 55 00 PM" src="https://user-images.githubusercontent.com/42228386/81520937-9c4d1c80-92fa-11ea-9020-61e80a46a66e.png">

Reply:
Done.